composer.json

```diff
@@ -1,36 +1,36 @@
 {
-  "name":        "league/tactician-container",
-  "description": "Tactician integration with the PHP League DI container",
-  "keywords":    ["tactician", "di", "container", "league"],
-  "license":     "MIT",
-  "authors": [
-    {
-      "name": "Nigel Greenway",
-      "homepage": "http://futurepixels.co.uk"
+    "name":        "league/tactician-container",
+    "description": "Tactician integration with the PHP League DI container",
+    "keywords":    ["tactician", "di", "container", "league"],
+    "license":     "MIT",
+    "authors": [
+        {
+            "name": "Nigel Greenway",
+            "homepage": "http://futurepixels.co.uk"
+        }
+    ],
+    "require": {
+        "php": ">=5.5",
+        "league/tactician": "^0.5@dev",
```

Comment:
Don't think this needs the @dev flag anymore now that 0.5 is just about out the door